tests/framework/Optimizers/GeneticAlgorithms/tests

```diff
@@ -375,15 +375,6 @@
     [../]
   [../]
 
-  [./GAwithEnsembleModelHDSMconvergence]
```

Comment:
why was this test removed?

Reply:
@Jimmy-INL I need your comments for this qustion.